ios/Framework/Core/Base/BaseListScreenlet/BaseListPageLoadInteractor.swift

```diff
@@ -18,16 +18,17 @@ public class BaseListPageLoadInteractor: ServerReadConnectorInteractor {
 
 	public let page: Int
 	public let computeRowCount: Bool
-
+    
 	public var resultAllPagesContent: [AnyObject?]?
 	public var resultPageContent: [AnyObject]?
 	public var resultRowCount: Int?
+    public var streamMode = false
```

Comment:
Configured your Xcode to use tabs, fix everything and add a new commit with that